### PR TITLE
Package bsmi_ocaml.0.1.3

### PR DESCRIPTION
**Pull request by cnmade**
### `bsmi_ocaml.0.1.3`
Bsmi ocaml library
A very useful tool collections

---
* Homepage: https://github.com/infobsmi/bsmi_ocaml
* Source repo: git+https://github.com/infobsmi/bsmi_ocaml.git
* Bug tracker: https://github.com/infobsmi/bsmi_ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+maintainer: ["cnmade"]
+authors: ["cnmade"]
+license: "AGPL-3.0-only"
+tags: ["topics" "tools" "your" "project"]
+homepage: "https://github.com/infobsmi/bsmi_ocaml"
+doc: "https://url/to/documentation"
+bug-reports: "https://github.com/infobsmi/bsmi_ocaml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.3"}
```
Comment (kit-ty-kate) [inline packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam:14]:
```
#=== ERROR while compiling bsmi_ocaml.0.1.3 ===================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://github.com/infobsmi/bsmi_ocaml/archive/0.1.3.tar.gz)
# path                 ~/.opam/4.02/.opam-switch/build/bsmi_ocaml.0.1.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p bsmi_ocaml -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/bsmi_ocaml-2517-b1a5e1.env
# output-file          ~/.opam/log/bsmi_ocaml-2517-b1a5e1.out
### output ###
# File "bin/dune", line 4, characters 23-25:
# 4 |  (libraries bsmi_ocaml re))
#                            ^^
# Error: Library "re" not found.
# -> required by _build/default/bin/bsmi_ocaml_hello.exe
# -> required by _build/install/default/bin/bsmi_ocaml
# -> required by _build/default/bsmi_ocaml.install
# -> required by alias install
```
```suggestion
  "dune" {>= "3.3"}
  "re"
```

File: packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bsmi ocaml library"
+description: "A very useful tool collections"
+maintainer: ["cnmade"]
+authors: ["cnmade"]
+license: "AGPL-3.0-only"
+tags: ["topics" "tools" "your" "project"]
+homepage: "https://github.com/infobsmi/bsmi_ocaml"
+doc: "https://url/to/documentation"
```
Comment (kit-ty-kate) [inline packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam:10]: This field is optional. If you don’t have documentation you can just remove this line

File: packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bsmi ocaml library"
+description: "A very useful tool collections"
```
Comment (kit-ty-kate) [inline packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam:4]: Could you try being a little more specific? What is Bsmi?

File: packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+maintainer: ["cnmade"]
+authors: ["cnmade"]
+license: "AGPL-3.0-only"
+tags: ["topics" "tools" "your" "project"]
```
Comment (kit-ty-kate) [inline packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam:8]: Those tags seem a bit suspicious. `tools` sure, why not, but `topics`, `your` and `project` seem very random

Comment (cnmade) [reply packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam:8]: This auto generate by dune. i will update it.

File: packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam (hunk 3 above)
Comment (cnmade) [reply packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam]: bsmi is just a name prefix, our OSS community has it's name , prefix with bsmi

**Comment by cnmade** [comment]
This is the initialize submit , and we will have test the whole submit process, and test with opam install and real project install . 
The full package will be continue improve continue and continue.

**Pull request closed without merging**